### DISCUSSION TITLE
Tycho should get a logo, your help is required!

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-tycho/tycho#1633] [Announcements]
Currently Tycho do not has a logo or any artwork and we like to change that so we can have something visual you can use on websites, blogs, talks, documentation that is identified with the project!

We plan to work together with a design studio and because of that here are some guiding question we would like to get answered from our users that can be used as input for an initial design:

1. What do you associate with Tycho e.g. building? packaging? maven? pde? integration? ... maybe even a picture or symbol?
2. What should the logo color associate? e.g. active, extraordinary, elegant, conservative, cool, technical, factual, modern etc.?
3. Which basic graphic shapes fit Tycho best? e.g. angular, pointed, roundish/symmetrical, roundish/organic, roundish/asymmetrical, spiky? 
4. Do you feel that Tycho Logo is connected to "Eclipse", thus should we try to incooperate [Eclipse Logos and Artwork](https://www.eclipse.org/org/artwork/) or should it stay independent?

Beside from that, it seems the original choice [for the name Tycho is lost](https://github.com/eclipse-tycho/tycho/discussions/1632), so please feel free to help discover why the name was choose or even express your own theory what the name "Tycho" means!

**Comment by akurtakov** [comment]
@jsievers You should be able to help with the name question :)

**Reply by jsievers** [reply]
Tycho was named by @jvanzyl 

I don‘t remember if there is any actual meaning either. As a physicist,  my association is Tycho Brahe and https://en.m.wikipedia.org/wiki/Tycho_(lunar_crater)


 Either way it sounds cool

**Reply by jvanzyl** [reply]
Named after Tycho the physicist.

**Comment by jonahgraham** [comment]
AFAIK The EF will pay for logos for projects and the board of directors approval is required for logos that incorporate the official eclipse logos. Let me know if you need more info on this as I don't have the information at hand right now.

**Reply by laeubi** [reply]
I think you mean this from the project handbook: https://www.eclipse.org/projects/handbook/#trademarks-project-logo ?
If yes, we are already in contact with the marketing team in this regard, but first like to gather some feedback from the community before we really start. For example if no one really feels the eclipse logo should be in cooperated in any way, we won't need to ask for clearance.

**Comment by HannesWell** [comment]
Personally I think the association to Eclipse should be reflected in the logo, since Tycho is all about building and packing Eclipse PDE plugin, features, products, etc.

If we want to stay in the cosmic/stellar theme maybe a stylized rocket could reflect that you can use Tycho to build/launch your 'cosmic' Eclipse product. :)

**Comment by jvanzyl** [comment]
Named after the physicist.jvz

> Tycho was named by @jvanzyl
> I don‘t remember if there is any actual meaning either. As a physicist,  my association is Tycho Brahe and https://en.m.wikipedia.org/wiki/Tycho_(lunar_crater)
> Either way it sounds cool

**Comment by laeubi** [comment]
Thanks for clarification maybe the schematic of the observatory build by Tycho Brahe could server as a good basis for a logo shape then!

- https://en.m.wikipedia.org/wiki/Stjerneborg 
- https://en.m.wikipedia.org/wiki/Uraniborg

Or probably one of his used instruments:

- https://en.m.wikipedia.org/wiki/Armillary_sphere

or the Tychonic system
- https://en.m.wikipedia.org/wiki/Tychonic_system

**Reply by HannesWell** [reply]
> Or probably one of his used instruments:
> 
>     * https://en.m.wikipedia.org/wiki/Armillary_sphere
> 

I think the Armillary sphere could look nice.
